docs/gettinginvolved/benchmarking.rst

```diff
@@ -0,0 +1,64 @@
+.. _benchmarking:
+
+Benchmarking
+============
+
+Flocker includes a tool for benchmarking operations.
+It needs the appropriate ssh-key added to a running ssh-agent.
+It is called like this:
+
+.. prompt:: bash $
+
+   python benchmark/benchmark_control.py <options>
```

Comment:
I guess we've gone for this approach (having the executable as a module in the package itself rather than a separate `bin` executable or an `entry_point`) because we don't want benchmarking distributed with the rest of flocker. Is that right?